### PR TITLE
add data files

### PR DESCRIPTION
**Pull request by shashank-reddy-code**

### CODE REVIEW

File: .devcontainer/Dockerfile [hunk 1]
```diff
@@ -1,5 +1,8 @@
 FROM mcr.microsoft.com/devcontainers/universal:2-linux
 
+WORKDIR /data
```
Comment (jakelee8) [inline .devcontainer/Dockerfile]: Add the data files to `/data`; they'll show up in the devcontainer automatically.

Comment (shashank-reddy-code) [reply .devcontainer/Dockerfile]: its already in `.devcontainer/data/`

Comment (shashank-reddy-code) [reply .devcontainer/Dockerfile]: is the copy redundant?

Comment (jakelee8) [reply .devcontainer/Dockerfile]: I mean move the data to the root of the repo

**Pull request merged**